doc/api_methods.md

```diff
@@ -402,7 +412,9 @@ Changes the opacity of the layer.
 
 #### Arguments
 
-+ **opacity**: value in range [0, 1]
+Arguments | Description
```

Comment:
"Arguments" is also the name of the section, so I'd use a better word to describe this column. Eg: name.

![screen shot 2015-11-04 at 11 42 49](https://cloud.githubusercontent.com/assets/390398/10935925/7a39d488-82e9-11e5-97b8-c01a69f8bf25.png)